core/language/en-GB/Buttons.multids

```diff
@@ -32,6 +32,8 @@ ExportTiddler/Caption: export tiddler
 ExportTiddler/Hint: Export tiddler
 ExportTiddlers/Caption: export tiddlers
 ExportTiddlers/Hint: Export tiddlers
+ExportTWCore/Caption: export tiddlywiki core
+ExportTWCore/Hint: Export tiddlywiki core for external-js wiki
```

Comment:
Perhaps "Export the ~TiddlyWiki core code for running with external ~JavaScript"